examples/integers/src/main.hs

```diff
@@ -0,0 +1,8 @@
+{-# LANGUAGE ForeignFunctionInterface #-}
+
+import Foreign.C.Types (CUInt(..))
```

Comment:
Is `CUInt` guaranteed to be 32-bits regardless of platform? In the other examples, I was trying to avoid using the "normal" `int` type, as it's only *guarantee* is that it is 16 bytes.